src/appmake/Makefile

```diff
@@ -11,7 +11,7 @@ else
 endif
 
 CC			?= gcc
-CFLAGS		+= -Wall -std=gnu11 -MMD -pedantic -I../common
+CFLAGS		+= -Wall -std=gnu11 -MMD -pedantic -I../common -lgmp -lssl -lcrypto
```

Comment:
These `-l` should be on LDFLAGS

Reply:
And for macOS -lssl -lcrypto aren't required.

Reply:
Given that we only use MD5, I'd be tempted to pull in the reference implementation as source code rather than have this extra dependency.

Reply:
I was thinking the same thing with MD5, as requiring the installing of openssl just for one target seems like a waste if it can be done another way.  With there was a good alternative with GMP though.